src/TLabel.h

```diff
@@ -43,17 +43,21 @@ Q_OBJECT
 
                   TLabel( QWidget * pW=0 );
 void              setScript( Host * pHost, const QString & func, const TEvent & args );
+void              setRelease( Host * pHost, const QString & func, const TEvent & args );
 void              setEnter( Host * pHost, const QString & func, const TEvent & args );
 void              setLeave( Host * pHost, const QString & func, const TEvent & args );
 void              mousePressEvent( QMouseEvent *  ) override;
+void              mouseReleaseEvent( QMouseEvent *  ) override;
 void              leaveEvent(QEvent *) override;
 void              enterEvent(QEvent *) override;
 
 QPointer<Host>    mpHost;
 QString           mScript;
+QString           mRelease;
 QString           mEnter;
 QString           mLeave;
 TEvent            mpParameters;
+TEvent            mrParameters;
```

Comment:
Oh, heh. This is what I mentioned on Gitter. I'm not very hip to naming conventions yet. :grimacing: I guess it's something I'll pick up over time. I can go through and fix every reference to mClickParameters in a commit so it's nice and clean everywhere.